client/components/token-field/index.jsx

```diff
@@ -39,7 +41,8 @@ var TokenField = React.createClass( {
 			saveTransform: function( token ) {
 				return token.trim();
 			},
-			onChange: function() {}
+			onChange: function() {},
+			tokenStatus: noop
```

Comment:
We could set an empty function like for `onChange` and avoid the `noop` dependency, or something like `_ => {}`